src/strategy/dungeons/wotlk/forgeofsouls/FosStrategy.cpp

```diff
@@ -0,0 +1,21 @@
+#include "FosStrategy.h"
+#include "FosMultipliers.h"
+
+void WotlkDungeonFosStrategy::InitTriggers(std::vector<TriggerNode*>& triggers) {
+    triggers.push_back(
+        new TriggerNode("move from bronjahm",
+                        NextAction::array(0, new NextAction("move from bronjahm", ACTION_MOVE + 5), nullptr)));
+    triggers.push_back(new TriggerNode(
+        "switch to soul fragment", NextAction::array(0, new NextAction("attack corrupted soul fragment", ACTION_RAID + 1), nullptr)));
+    triggers.push_back(new TriggerNode("bronjahm position",
+        NextAction::array(0, new NextAction("bronjahm tank position", ACTION_RAID + 1),
```

Comment:
These should probably not be queued like this, and tank target is not really a necessary action, you can do dps + tank + heal targeting in a single action